js/rooms.js

```diff
@@ -45,7 +45,14 @@ $(document).ready(function() {
 				success: function(data) {
 					$('#app-navigation ul').html('');
 					data.forEach(function(element) {
-						$('#app-navigation ul').append('<li><a href="#'+escapeHTML(element['id'])+'">'+escapeHTML(element['name'])+' <span class="utils">' + escapeHTML(element['count']) + '</span></a></li>');
+						$('#app-navigation ul').append(
```

Comment:
@ChristophWurst so I assume we should get in #60 first? We also need to coordinate with @LukasReschke’s #55 person selector.